src/createSlice.ts

```diff
@@ -259,9 +259,7 @@ export function createSlice<
     actionMatchers = [],
     defaultCaseReducer = undefined
   ] =
-    typeof options.extraReducers === 'undefined'
-      ? []
-      : typeof options.extraReducers === 'function'
+    typeof options.extraReducers === 'function'
```

Comment:
If options.extraReducers === undefined, then the result would be [undefined], right?

Reply:
It's a little "wrap-your-head-around" with this.

for `options.extraReducers === undefined`, this would result in 
```js
 const [ 
   extraReducers = {}, 
   actionMatchers = [], 
   defaultCaseReducer = undefined 
 ] = [undefined]
```
which would then take on the default value of `extraReducers = {}` at desctructuring.

So yeah, I believe the PR is correct, but I'd like a second opinion here @markerikson or @msutkowski

Reply:
Let's go with it.